core/src/main/java/org/apache/gravitino/storage/relational/mapper/provider/base/UserRoleRelBaseSQLProvider.java

```diff
@@ -28,8 +28,7 @@
 
 public class UserRoleRelBaseSQLProvider {
 
-  public String batchInsertUserRoleRel(
-      @Param("userRoleRelList") List<UserRoleRelPO> userRoleRelList) {
+  public String batchInsertUserRoleRel(@Param("userRoleRels") List<UserRoleRelPO> userRoleRels) {
```

Comment:
Can you add some UTs or ITs to cover the change to avoid similar problems in the future?